cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/co/cask/cdap/templates/etl/batch/BatchETLDBAdapterTest.java

```diff
@@ -71,18 +67,17 @@
 import java.sql.Types;
 import java.text.SimpleDateFormat;
 import java.util.List;
-import java.util.Map;
 import java.util.concurrent.TimeUnit;
 import javax.sql.rowset.serial.SerialBlob;
 
 /**
  * Test for ETL using databases
  */
-public class BatchETLDBAdapterTest extends TestBase {
+@Ignore
```

Comment:
right now it is ignored because the framework needs some more work to support it, since it does my hypersql setup.